### PR TITLE
docs: rm type hints in ignite.utils (1)

### PR DESCRIPTION
**Pull request by ydcjeff**
Part of #1665 

Description:
- remove type hints in docstring of ignite.utils
- changed input_ to x as `input_:` is parsed as link in rst

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [x] Documentation is updated (if required)

### CODE REVIEW

File: ignite/utils.py [hunk 1]
```diff
@@ -12,53 +11,71 @@
 
 
 def convert_tensor(
-    input_: Union[torch.Tensor, collections.Sequence, collections.Mapping, str, bytes],
+    input: Union[torch.Tensor, collections.Sequence, collections.Mapping, str, bytes],
```
Comment (vfdev-5) [inline ignite/utils.py:14]: Let's remove built-in keyword `input` by `x`

File: ignite/utils.py [hunk 2]
```diff
@@ -12,53 +11,71 @@
...
     def _func(tensor: torch.Tensor) -> torch.Tensor:
         return tensor.to(device=device, non_blocking=non_blocking) if device is not None else tensor
 
-    return apply_to_tensor(input_, _func)
+    return apply_to_tensor(input, _func)
 
 
 def apply_to_tensor(
-    input_: Union[torch.Tensor, collections.Sequence, collections.Mapping, str, bytes], func: Callable
+    input: Union[torch.Tensor, collections.Sequence, collections.Mapping, str, bytes], func: Callable
```
Comment (vfdev-5) [inline ignite/utils.py:34]: same here

File: ignite/utils.py [hunk 3]
```diff
@@ -12,53 +11,71 @@
...
+        input: input tensor or mapping, or sequence of tensors.
+        func: the function to apply on ``input``.
     """
-    return apply_to_type(input_, torch.Tensor, func)
+    return apply_to_type(input, torch.Tensor, func)
 
 
 def apply_to_type(
-    input_: Union[Any, collections.Sequence, collections.Mapping, str, bytes],
+    input: Union[Any, collections.Sequence, collections.Mapping, str, bytes],
```
Comment (vfdev-5) [inline ignite/utils.py:46]: here as well

**Review by vfdev-5 — COMMENTED** [review]
Thanks @ydcjeff ! A nit comment

**Comment by ydcjeff** [comment]
If this looks good, this can be merged first.

**Review by vfdev-5 — APPROVED** [review]
LGTM! Thanks @ydcjeff !

**Comment by ydcjeff** [comment]
I have added number in my opened PRs that can be merged one after another.

**Pull request merged**